src/App.css

```diff
@@ -61,6 +61,9 @@ form{
   border-bottom: 2px solid #fff !important;
   border-radius: 0 !important;
 }
+.custom-input::placeholder {
+  color: rgb(239, 228, 228) !important; /* Set the desired color */
```

Comment:
this is good, but the placeholder text should be lighter than the text around it and it shouldn't be the same as the input text
take this example of the Facebook Login :
![image](https://github.com/Rawan-Mohamed/project-management-system-1/assets/64327685/03d4bbb0-9ee1-4bc3-b791-2ff55cb76344)
![image](https://github.com/Rawan-Mohamed/project-management-system-1/assets/64327685/d9867da6-7fc7-49e3-bbeb-8914eac26963)